plugins/woocommerce-admin/client/marketing/components/product-icon/icons/index.js

```diff
@@ -29,6 +31,21 @@ export { default as mailchimpForWoocommerce } from './library/mailchimp';
 // MailPoet
 export { default as mailpoet } from './library/mailpoet';
 
+// Creative Mail by Constant Contact
+export { default as creativeMailByConstantContact } from './library/creative-mail';
+
+// Jetpack CRM
+export { default as zeroBsCrm } from './library/jetpack-crm';
```

Comment:
Why do we export it as `zeroBsCrm`, instead of `jestpackCrm`?  🤔

Reply:
Jumping into this PR... `zero-bs-crm` is the slug for Jetpack CRM in the WPORG repo, and only that. For the rest of things outside of the WPORG repo, like variables, files name, folders, comments or mentions should use the new brand, Jetpack CRM, or derivates like `jpcrm` `jetpack-crm` `jetpackcrm`.

Reply:
@cleacos Gotcha, thanks for the context! 🙂